installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -45,7 +46,11 @@ include::modules/installation-initializing.adoc[leveloffset=+1]
 
 include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
 include::modules/installation-osp-config-yaml.adoc[leveloffset=+2]
+//Dual-stack networking
+include::modules/install-osp-dualstack.adoc[leveloffset=+2]
+include::modules/install-osp-deploy-dualstack.adoc[leveloffset=+3]
```

Comment:
I don't think we need to include this docs for UPI. This docs are meant for IPI.
If the user wants to enable a dual-stack installation with UPI the only thing needed is to uncomment the `os_subnet6` in [the inventory file](https://github.com/openshift/installer/blob/master/upi/openstack/inventory.yaml#L77).